src/Forms/GroupForm/GroupForm.jsx

```diff
@@ -159,6 +165,15 @@ const GroupForm = ({
           }}
           value={name}
         />
+        <Textarea
+          placeholder={t("screens.allGroups.groupDescription")}
+          onChange={(value) => {
+            setDescription(value);
+          }}
+          value={description}
+          rows={5}
+          maxlength={250}
```

Comment:
Can Webadresse this to the constants file so we can use this also in the other forms with a Description?

Reply:
Done